resources/lang/en/fieldtypes.php

```diff
@@ -58,6 +58,7 @@
     'markdown.config.parser' => 'The name of a customized Markdown parser. Leave blank for default.',
     'markdown.config.restrict' => 'Prevent users from navigating to other folders.',
     'markdown.config.smartypants' => 'Automatically convert straight quotes into curly quotes, dashes into en/em-dashes, and other similar text transformations.',
+    'radio.config.cast_booleans' => 'Options with values of true and false will be saved as booleans.',
```

Comment:
For bonus points, since it's exactly the same as the option in the select fieldtype, you could:

- Delete this key.
- Move `select.config.cast_booleans` to `any.config.cast_booleans` in all the language files.
- Adjust the translation keys in both fieldtypes.

Reply:
Sure!